src/App.tsx

```diff
@@ -1,9 +1,16 @@
 import React from "react";
+import { Server } from "miragejs"
+
+import mocksConfig from "./mocks";
 import Sidebar from "./components/sidebar/Sidebar";
 import Header from "./components/Header";
 import Studio from "./components/studio/Studio";
 import "./App.scss";
 
+if (process.env.NODE_ENV !== 'production') {
```

Comment:
we can move this to header may be as we also need to check if the app is running inside JHonline